tests/Pager_Test.php

```diff
@@ -34,35 +35,61 @@ public function setUp(): void
         $this->pager = new \Recurly\Pager($client_stub, 'page_one');
     }
 
+    public function testMapArrayParams(): void
+    {
+        $ids = ['id-1', 'id-2'];
+        $idsCsv = join(rawurlencode(','), $ids);
+        $url = "https://v3.recurly.com/resources?ids=$idsCsv";
```

Comment:
I think instead of doing `join(rawurlencode(','), $ids);` on the assertion, it's better to spell out in string form what you expect. This is duplicates the code that you are trying to test.

Reply:
Agreed, updated the PR